src/mixins/nr-affiliation-mixin.ts

```diff
@@ -113,10 +117,10 @@ export class NrAffiliationMixin extends Mixins(CommonMixin) {
 
     return {
       filing: {
-        header: { name, accountId },
         business: { legalType },
-        registration: this.isFirm(nr) ? { nameRequest: { legalType, nrNumber } } : undefined,
-        incorporationApplication: !this.isFirm(nr) ? { nameRequest: { legalType, nrNumber } } : undefined
+        header: { accountId, name },
+        incorporationApplication: !this.isFirm(nr) ? { nameRequest: { legalType, nrNumber } } : undefined,
+        registration: this.isFirm(nr) ? { nameRequest: { legalType, nrNumber } } : undefined
```

Comment:
Here, I simply reordered the properties to match the (sorted) properties in the JSON (eg, in the network tab).